user-guide/Troubleshooting/Known_issues/KI_DataMinerMessageBroker.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: KI_DataMinerMessageBroker
```

Comment:
very generic KI title, might be better to specify further to prevent confusion if more come in cfr MessageBroker since it's used quite extensively.

example: KI_DataMinerMessageBroker_Chunking_MaxPayload

Reply:
Done in commit [e705760](https://github.com/SkylineCommunications/dataminer-docs/pull/1995/commits/e705760295b3bb2b7ab0b049adc593b99a1898f7)